contracts/mirror_gov/src/migrate.rs

```diff
@@ -153,7 +153,7 @@ pub fn migrate_state<S: Storage>(storage: &mut S) -> StdResult<()> {
     Ok(())
 }
 
-pub fn migrate_polls<S: Storage>(storage: &mut S) -> StdResult<()> {
+pub fn migrate_polls<S: Storage>(storage: &mut S, env: Env) -> StdResult<()> {
```

Comment:
we also need to migrate these three periods to height unit.
```
voting_period: legacy_config.voting_period,        
effective_delay: legacy_config.effective_delay,       
expiration_period: legacy_config.expiration_period,
```

Reply:
added requested changes